contracts/test/ComputeSlotTests.t.sol

```diff
@@ -0,0 +1,20 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {Test} from "forge-std/Test.sol";
+// solhint-disable no-console
+import {console2} from "forge-std/console2.sol";
+
+contract ComputeSlotTest is Test {
```

Comment:
Thoughts on making the storage slot variables public for the sake of both making them easy to unit test?